programs/programs/co/rtdlive/tests/test_qualifying_relative_impact.py

```
@@ -0,0 +1,130 @@
+        return Screen.objects.create(
+            white_label=self.white_label,
+            completed=False,
+            last_tax_filing_year="2024",
+            zipcode=zipcode,
+        )
+
+    def _make_calculator(self, screen: Screen) -> RtdLive:
+        return RtdLive(screen, _make_program(), {}, _make_missing_deps())
+
+    def test_eligible_when_household_income_is_low(self):
+        """Parents $30k + adult child $0 → combined $30k < 3-person 2.5×FPL ($64,550) → eligible."""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)